website/public/locale/en/translation.yaml

```diff
@@ -281,26 +281,27 @@ contributing:
 
 adoptersTestimonials:
   rejectDataInfo: Unable to fetch data correctly
-  becomeAdopter: Become an adopter
+  becomeAdopter: Become an Adopter
+
 
 # Blog
 
-blog:
-  title: Community contributed guides and blogs
-  readMore: Read more
-  read: Read
-  articles: All articles
-  all: All
-  chaosengineering: chaosengineering
-  devops: devops
-  openebs: openebs
-  solutions: solutions
-  tutorials: tutorials
-  minToRead: min to read
-  share: Share
-  greetings: Great news! You’re 1 article smarter!
-  previousArticle: Previous article
-  nextArticle: Next article
-  recommendationsTitle: Based on your reading story
-  blog: Blog
-  readTime: read time
+# blog:
```

Comment:
This may not be the right place to disable the sections. These are translator text messages (i18n).